lib/api/rule.go

```diff
@@ -24,7 +24,8 @@ import (
 )
 
 type Rule struct {
-	Action string `json:"action" validate:"action"`
+	Action    string  `json:"action" validate:"action"`
+	LogPrefix *string `json:"logPrefix,omitempty" validate:"omitempty"`
```

Comment:
If the action is "log", but the LogPrefix is nil, is there a default value, or just no prefix (null in etcd)?